iceoryx_posh/test/moduletests/test_capro_message.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iceoryx_posh/internal/capro/capro_message.hpp"
+#include "iceoryx_posh/internal/roudi/introspection/port_introspection.hpp"
+#include "test.hpp"
+
+using namespace ::testing;
+using namespace iox::capro;
+
+class CaproMessage_test : public Test
+{
+  public:
+    CaproMessage_test()
+    {
+    }
+
+    virtual ~CaproMessage_test()
+    {
+    }
+
+    virtual void SetUp(){};
+    virtual void TearDown(){};
+};
+
+
+TEST_F(CaproMessage_test, CTorSetsParametersCorrectly)
+{
+    uint16_t testServiceID = 1;
```

Comment:
These three could be constexpr, like so```constexpr uint16_t testServiceId{1}``` This leads to (slight) optimization potential. General rule: try to make const or constexpr what is possible. Note that in this case I would not advocate to use CAPITAL_LETTER_STYLE for such constexprs (only for constants with a large scope, e.g. at class scope), i.e. leave the names as is.

Reply:
okay